samples/backend/python/quart/model/model_utils.py

```diff
@@ -0,0 +1,85 @@
+from dataclasses import fields, is_dataclass
+from typing import Type, TypeVar, Union, get_args, get_origin, get_type_hints
+from enum import Enum
+
+T = TypeVar("T")
+
+
+def rename(original_name: str, target_name: str):
+    def wrapper(cls: Type[T]) -> Type[T]:
+        renames = getattr(cls, "__renames__", {})
+        renames[original_name] = target_name
+        setattr(cls, "__renames__", renames)
+        return cls
+
+    return wrapper
+
+
+def serializable(cls: Type[T]) -> Type[T]:
```

Comment:
Can we get this for free if we used Pydantic models perhaps? Am surprised to see we need all this code.

Reply:
yeah, we can move to use pydantic, we wouldn't need all of this with that, I wasn't really aware of that library :)